packages/bcer-api/app/src/cron/config/cron.config.ts

```diff
@@ -0,0 +1,26 @@
+export class CronConfig {
+    private static closeLocationCronTime: string = process.env.CLOSE_LOCATION_CRON_TIME || '0 3 15 1 *';
+    private static noiExpiryCronTime: string = process.env.NOI_EXPIRY_CRON_TIME || '0 3 1 12 *';
+    private static runCronFlag: boolean = process.env.RUN_CRON === 'true';
+    private static cronTimeZone: TimeZone = { timeZone: process.env.CRON_TIME_ZONE || 'America/Vancouver' };
+  
+    static getCloseLocationCronTime(): string {
+      return this.closeLocationCronTime;
+    }
+  
+    static getNoiExpairyCronTime(): string {
+      return this.noiExpiryCronTime;
+    }
+  
+    static getRunCronFlag(): boolean {
```

Comment:
I'm wondering if we maybe should change this to toggle based on the name of the cron job? That way we can have a way of disabling individual cron jobs if needed